### PR TITLE
added remaining role entity syncs

### PR DESCRIPTION
**Pull request by kti-sam**

### CODE REVIEW

File: djautotask/models.py [hunk 1]
```diff
@@ -565,3 +565,35 @@ class Department(models.Model):
 
     def __str__(self):
         return self.name
+
+
+class ResourceRoleDepartment(models.Model):
```
Comment (kti-cameron) [inline djautotask/models.py]: I just noticed that these are not `TimeStampedModel`'s like most of the other models. Probably doesn't really matter but any reason why?

Comment (kti-sam) [reply djautotask/models.py]: Do these models need any of the methods or attributes of a `TimeStampedModel`? I didn't think they did, so I did not include it. I thought the those were used for dealing with partial syncs and querying in CW and AT or something, so we didn't need them on models with no `last_updated_field`. Let me know if I am wrong about that.

Comment (kti-cameron) [reply djautotask/models.py]: All `TimeStampedModel` provides is created and modified fields. I could be mistaken but I don't think we actually use the created or modified fields in the synchronizers. And the `last_updated_field` is a field on the model so is not related to the created or modified fields. 

Anyway, I figured it was worth asking. Seeing as we don't appear to really need them right now. We can always add it later if we do need it.

**Comment by kti-cameron** [comment]
Looks good. Merging

**Pull request merged**